src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -300,4 +325,28 @@ public void resetComments(String project, String repository, String pullRequestI
       LOGGER.debug("Exception stack trace", e);
     }
   }
+  
+  /**
+   * Get available severity to create a task.
+   */
+  public List<String> getAvailableSeverities() {
+      List<String> result = new ArrayList<>();
+      String threshold = config.getTaskIssueSeverityThreshold();
+        
+      // threshold == NONE, no severities available 
+      if (! StringUtils.equals(threshold, StashPlugin.SEVERITY_NONE)) {
+        
+        // INFO, MINOR, MAJOR, CRITICAL, BLOCKER
+        boolean hit = false;
+        for (String severity : StashPlugin.SEVERITY_LIST) {
+          
+          if (hit || StringUtils.equals(severity, threshold)) {
```

Comment:
Possible discrepancy: while the severity is not matched, then we add the severity to result and we carry on.
For example if it's MAJOR, then we should have result to [INFO,MINOR,MAJOR]
Proposal:
// INFO, MINOR, MAJOR, CRITICAL, BLOCKER
        for (String severity : StashPlugin.SEVERITY_LIST) {
          result.add(severity);
          if (StringUtils.equals(severity, threshold)) {
            break;
          }
        }